lib/lootnscoot/loot_lib.lua

```diff
@@ -110,17 +110,18 @@ There is also no flag for combat looting. It will only loot if no mobs are withi
 
 ]]
 
-local mq = require 'mq'
+local mq           = require 'mq'
 
-local eqServer = string.gsub(mq.TLO.EverQuest.Server(), ' ', '_')
+local eqServer     = string.gsub(mq.TLO.EverQuest.Server(), ' ', '_')
 -- Check for looted module, if found use that. else fall back on our copy, which may be outdated.
-local eqChar = mq.TLO.Me.Name()
-local actors = require('actors')
+local eqChar       = mq.TLO.Me.Name()
+local RGMercUtils  = require("utils.rgmercs_utils")
+local actors       = require('actors')
```

Comment:
RGMercUtils already imports and makes available actors don't really need to import it again.

Reply:
ahh good call